### DISCUSSION TITLE
Show errors on the line number or something more than the red squiggly

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1133] [Ideas]
**Is your feature request related to a problem? Please describe.**
When I have an extra parenthesis, or have typed a period instead of a comma, or other single character issue, the red squiggly under that _single character_ is very difficult to find.

See this code:
![image](https://github.com/TabularEditor/TabularEditor3/assets/45298358/ff506fa7-0803-40c9-b0ff-1477f2ace548)

**Describe the solution you'd like**
In the image above, the error is on line 51. It would be nice if the 51 in the UI were red, reverse highlighted, or something. A much larger callout.

**Describe alternatives you've considered**
None. I just keep searching for the red squiggly.

**Comment by mlonsk** [comment]
Hi @edhans 
Thank you for the suggestion, I can definitely see how it is difficult to see such small errors. 
I have added your idea to our backlog.

The way to handle this currently is to use the Messages window, as it shows you the line of the error, and you can double click on it to go the line in question:
![2023-08-28_09-24-39](https://github.com/TabularEditor/TabularEditor3/assets/11939114/b29f449a-9100-4f3c-b579-224cde56362d)

**Comment by edhans** [comment]
Ahhh... yes. I don't always remember to look at the messages window. Most of the time it is visible, I just don't notice when working on a measure, but it isn't always visible. I may have the Expression window at full screen for example.

But yes, I will try to remember to use it because it definitely points to the correct line.

**Comment by mlonsk** [comment]
Hi @edhans 
In Tabular Editor 3.19.0, we have added scrollbar indicators for errors, warnings, and code actions. Does this fulfill the requirement you expressed in this feature request, or do you feel the need for more visibility remains?

**Reply by edhans** [reply]
Probably good enough. I just need to train myself to look at the scroll bar area even when there is no need to scroll (short measure) to see these indicators.

Thanks!
